package.json

```diff
@@ -130,7 +130,7 @@
     "react-dom": "^16.4.0",
     "react-final-form": "^4.1.0",
     "react-redux": "^5.0.6",
-    "react-select": "^2.4.2",
+    "react-select": "^3.0.4",
```

Comment:
Did you made sure that nothing else broke? There are form components using this dependency.

Reply:
I double-checked: none of the select boxes had breaking changes and all of them was working (however there are some components which were broken before)